lib/common/imageTool.py

```diff
@@ -51,8 +52,11 @@ def convert_video_to_images(self, input_video_fp, output_image_dir_path, output_
             os.mkdir(output_image_dir_path)
             while result:
                 str_image_fp = os.path.join(output_image_dir_path, "image_%d.jpg" % img_cnt)
-                if (img_cnt >= self.search_range[0] and img_cnt <= self.search_range[1]) or (img_cnt >= self.search_range[2] and img_cnt <= self.search_range[3]):
+                if comp_mode and img_cnt >= self.search_range[0]:
                     cv2.imwrite(str_image_fp, image)
+                else:
+                    if (img_cnt >= self.search_range[0] and img_cnt <= self.search_range[1]) or (img_cnt >= self.search_range[2] and img_cnt <= self.search_range[3]):
+                        cv2.imwrite(str_image_fp, image)
```

Comment:
hmm, if these two condition branches doing the same thing, i'll suggest to merge it into one condition.
ex:
```python
if (comp_mode and img_cnt >= self.search_range[0]) or \
    (img_cnt >= self.search_range[0] and img_cnt <= self.search_range[1]) or \
    (img_cnt >= self.search_range[2] and img_cnt <= self.search_range[3]):
    cv2.imwrite(str_image_fp, image)
```